deploy/kubernetes/operator.yaml

```diff
@@ -12,6 +16,9 @@ spec:
     metadata:
       labels:
         name: nsx-ncp-operator
+        tier: nsx-networking
```

Comment:
we must Add "tier: nsx-networking" labels to spec.metadata so that these labels could be applied to pod deployment.